ui/js/controllers/perf/alerts.js

```diff
@@ -19,16 +19,55 @@ perf.factory('PhAlerts', ['$http', 'thServiceDomain', function($http, thServiceD
             return $http.get(href).then(function(response) {
                 return response.data;
             });
+        },
+        reassignAlertSummary: function(alertId, revisedSummaryId) {
+            return $http.put(thServiceDomain +
+                             '/api/performance/alert/' + alertId + '/',
+                             { revised_summary_id: revisedSummaryId });
         }
     };
 }]);
 
+perf.controller(
+    'ReassignAlertsCtrl',
+    function($scope, $modalInstance, $http, $q, alertSummary, PhAlerts) {
+        var alerts = _.where(alertSummary.alerts, {'selected': true});
+        $scope.reassignAlerts = function() {
+            var revisedSummaryId = parseInt(
+                $scope.reassignAlertId.newAlertSummaryId.$modelValue);
```

Comment:
Though to be fair, the UI doesn't have any error handling right now. We need to fix that, but I'm not sure if it should block landing this.